tools/protos/test_status.proto

```diff
@@ -0,0 +1,118 @@
+// Copied from `external/@bazel_tools/src/main/protobuf/test_status.proto`.
+// https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/test_status.proto.
```

Comment:
Lets use the permalink to the exact version we are using since it is tagged on github
```suggestion
// https://github.com/bazelbuild/bazel/blob/4.2.1/src/main/protobuf/test_status.proto
```